src/App.tsx

```diff
@@ -1,31 +1,47 @@
-import React from 'react';
+import { ChangeEvent, FC, useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-export const App: React.FC = () => {
+export const App: FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter(({ title, description }) => {
+    const lowerCasedTitle = title.toLowerCase();
+    const lowerCasedQuery = query.toLowerCase().trim();
+    const lowerCasedDescription = description.toLowerCase();
+
+    return lowerCasedTitle.includes(lowerCasedQuery)
+      || lowerCasedDescription.includes(lowerCasedQuery);
```

Comment:
you share the same logic `toLowerCase`, `includes` between title and description comparison, create helper func for it